packages/launcher/src/widget.tsx

```diff
@@ -174,8 +174,11 @@ export class Launcher extends VDomRenderer<ILauncher.IModel> {
                 icon={icon}
                 iconClass={classes(iconClass, 'jp-Icon-cover')}
                 stylesheet="launcherSection"
-              />
-              <h2 className="jp-Launcher-sectionTitle">{cat}</h2>
+                title={cat + ' icon'}
+              />{' '}
+              <h2 className="jp-Launcher-sectionTitle" title={cat + ' title'}>
```

Comment:
- we should not concatenate strings as this prevents internationalisation; all strings should be translated;
- I think there is no need for title on `h2` here, this does not add any information

```suggestion
                title={this._trans.__('%s icon', cat)}
              />{' '}
              <h2 className="jp-Launcher-sectionTitle">
```